libs/wingsdk/src/target-sim/bucket.inflight.ts

```diff
@@ -17,21 +18,25 @@ import {
 
 export class Bucket implements IBucketClient, ISimulatorResourceInstance {
   private readonly objectKeys: Set<string>;
-  private readonly fileDir: string;
+  private readonly _fileDir: string;
   private readonly context: ISimulatorContext;
   private readonly initialObjects: Record<string, string>;
   private readonly _public: boolean;
   private readonly topicHandlers: Partial<Record<BucketEventType, string>>;
 
   public constructor(props: BucketSchema["props"], context: ISimulatorContext) {
     this.objectKeys = new Set();
-    this.fileDir = fs.mkdtempSync(join(os.tmpdir(), "wing-sim-"));
+    this._fileDir = fs.mkdtempSync(join(os.tmpdir(), "wing-sim-"));
     this.context = context;
     this.initialObjects = props.initialObjects ?? {};
     this._public = props.public ?? false;
     this.topicHandlers = props.topics;
   }
 
+  public get fileDir(): string {
+    return this._fileDir;
+  }
```

Comment:
I don't think this getter is used anywhere, so we can probably remove it?

Reply:
Oops, never mind I see

Reply:
Yeah, I added that getter because one of the tests was bypassing TypeScript checks and calling a private method, which I didn't like.

Reply:
Yeah, I added that getter because one of the tests was bypassing TypeScript checks and calling a private property, which I didn't like.